server/src/main/kotlin/com/xebia/functional/xef/server/services/RepositoryService.kt

```diff
@@ -20,3 +24,6 @@ object RepositoryService {
     }
 
 }
+
+fun String.getUser() =
```

Comment:
I like the idea, but in this case, I would say that makes more sense to wrap the String value in a Token class or something similar. I don't know if is good to add a function to a primitive type which depends on a BD to work.

Reply:
Sure!